docs/meetings/client/2022_10_07.md

```diff
@@ -0,0 +1,119 @@
+
+2. NVIDIA (Audio to face) neural net takes audio source and can sync it to any 3D object. Probably supported for a while, but it will require licensing fees that are not clear. Probably resource-heavy because it requires processing power. Result looks really good. Processing time will probably be heavy as well
+
+3. Lazy KH approach (repo name), maps audio to phonemes and mouth expression. Doesn’t use any kind of deep learning or machine learning. Least computationally expensive. Not uploading an image, so it’s difficult to upload an avatar, but you can have some control (upload a base avatar). User can upload different images and textures, face avatar and mouths and eyes, and then can upload audio and text, and the approach will animate the mouth speaking on the avatar. More difficult with an upload. Very simple to implement, very little processing power, only available on Mac, from 2015, and it’s slow – one-off project but they could optimize it if they take it and customize it
+
+- Nazim: Timing? Time to compute audio to face animations is very quick, but the problem is you need high-end hardware with good computing power. Resources would be quite high.
+- Erik: Micro or medium resources currently, nothing insane. The NVIDIA solution might be too heavy. Can always increase computing power or isolate it with an API, Kukarella would need to weigh the options.
+ 
+# (16:45) QUESTIONS FOR KUKERELLAS TECH
+
+- Erik:
+    - would like a small written-up report to review in more detail
+    - ach team could try a different tech stack
+
+- Nazim:
+    - Some research solutions can come from unexpected places
+    - Let’s try different approaches
+
+# Student Questions
+
+- Audio file format: MP3 and WAV files produced
+- Video file format preferred: MOV and MP4
+
+Conclusion:
+-	TEAM A: We prefer LipGAN
+-	Yash’s team: not sure what they said?
+-	TEAM C: Split between Wav2Lip and Audio2Face, perhaps making an alternate version of it
+
+Next week:
+-	Each team chooses what to do (the model/approach) and goes more in-depth requirement and information about where to go from that point
+-	Build our app as an API (they have an API server on Node.js on Amazon EC2), JavaScript is good as well
+-	If python, then can run an external server for that
+-	They will want an API of some kind
+-	Their current system does not currently create the timestamps, but they’re working on subtitles
+-	The phoneme extraction is all for us to do
+-	Access to the SVG images for the avatars on table reads
+-	Also have some characters from their mobile apps with some animation available
+-	Nazim has no preference between 2D and 3D animation, whichever is more realistic and faster to process
+
+TODO:
+-	Share repositories
+-	Request access to API
+-	Create an account with an email address and then send the email address to them to receive the Kukarella prime account
+Nazim’s research: Google Imagen? See link in Discord chat
```

Comment:
I don't think this was a research thing - more of an aside to point out some crazy results that can be achieved